nixos/tests/systemd-portabled.nix

```diff
@@ -0,0 +1,56 @@
+import ./make-test-python.nix ({pkgs, lib, ...}: let
+  systemdWithPortabled = pkgs.systemd.override {
+    withPortabled = true;
+  };
```

Comment:
```suggestion
```
This is not really required since portabled is on by default.

Reply:
Ah yes, well it wasn't at the start of this PR.  Didn't remove it when I set the default to true, because I wanted to find out how ofborg reacts to a change needing sooo many rebuilds :)